tasks/update-cmr-access-constraints/index.js

```diff
@@ -44,16 +73,12 @@ function setRestrictionMetadataInEcho10XMLMetadata(
   echo10XMLMetadataContentsObject,
   accessConstraintsObject
 ) {
-  const { description, value } = accessConstraintsObject;
-  const echo10AccessConstraintsObject = {
-    RestrictionComment: description !== undefined ? description : 'None',
-    RestrictionFlag: value,
-  };
-  const updatedGranule = Object.assign(
-    echo10AccessConstraintsObject,
-    echo10XMLMetadataContentsObject.Granule
+  const metadataGranule = echo10XMLMetadataContentsObject.Granule;
+  const updatedMetadataGranule = createUpdatedEcho10XMLMetadataGranuleCopy(
+    metadataGranule,
+    accessConstraintsObject
```

Comment:
Do we do this in UMM too? I haven't dug around for it.

Reply:
Oh but we update UMM right?

Reply:
[Yes](https://github.com/nasa/cumulus/pull/1840/files#diff-084dd377757f06750bfb541b88663f98R84).